contracts/interchain-token-service/src/state.rs

```diff
@@ -38,6 +46,29 @@ pub fn save_config(storage: &mut dyn Storage, config: &Config) -> Result<(), Err
     Ok(CONFIG.save(storage, config)?)
 }
 
+pub fn load_chain_config(
+    storage: &dyn Storage,
+    chain: &ChainNameRaw,
+) -> Result<Option<ChainConfig>, Error> {
+    Ok(CHAIN_CONFIGS.may_load(storage, chain)?)
+}
+
+pub fn save_chain_config(
+    storage: &mut dyn Storage,
+    chain: &ChainNameRaw,
+    max_uint: Uint256,
+    max_target_decimals: u8,
+) -> Result<(), Error> {
+    Ok(CHAIN_CONFIGS.save(
```

Comment:
Prevent overwriting in this method so it can't be misused, similar to `register_its_contract`

Reply:
I don't like the idea of embedding business logic in I/O methods.

Reply:
I actually also prefer preventing overwriting here. Doesn't feel like business logic to me, but rather a state invariant, which I think is appropriate for this module to handle.

Reply:
The state being not able to overwrite is business logic though.